package.json

```diff
@@ -63,6 +61,9 @@
     "ie 11"
   ],
   "dependencies": {
-    "dequal": "^2.0.3"
+    "deep-equal-json": "^1.0.0"
+  },
+  "engines": {
+    "node": ">= 0.4"
```

Comment:
Why does it list engines as `node >= 0.4` but only test `>= 0.8` in the tests? Seems counter-intuitive to the spirit of the PR.